src/lib/slider/slider.scss

```diff
@@ -29,6 +29,11 @@ $mat-slider-focus-ring-size: 30px !default;
 
 
 .mat-slider {
+  &:not(.mat-slider-disabled):active,
+  &.mat-slider-sliding:not(.mat-slider-disabled) {
+    @include cursor-grabbing;
```

Comment:
There are some similar styles down here: https://github.com/angular/material2/pull/14614/files#diff-272317cfcd69bbb18560d47fa687cb15R114. Can they be removed?